RDoc

alexsnaps opened the pull request:
Added some (what I think may well be valid) documentation to this thing

Commit:
RDoc

maximebedard commented on ext/gctrack/gctrack.c:
```diff
@@ -100,6 +121,26 @@ gctracker_start_record(int argc, VALUE *argv, VALUE klass)
+ * Ends the recording of the last started record and return the 
+ * the collected information. 
+ * The array contains two numbers: The amount of GC cycles observed and 
+ * the cumulative duration of these cycles. If multiple recordings were started
```
Worth nothing that the duration is captured in nanosec?

alexsnaps replied:
s/duration/duration in ns/

csfrancis replied:
May want to specify the units for the duration.

maximebedard commented on ext/gctrack/gctrack.c:
```diff
@@ -118,12 +159,35 @@ gctracker_end_record(int argc, VALUE *argv, VALUE klass)
+ * call-seq:
+ *    GC::Tracker.enable   -> true or false
+ *
+ * Returns +true+ if the Tracker has been enabled, +false+ otherwise.
```
I believe the description here needs to be swapped with the previous function.

Commit:
README

Commit:
Doc fixes

alexsnaps commented:
Added the [README.md](https://github.com/Shopify/gctrack/blob/75248574202d165add0d05d55b821ad3d9d45b82/README.md)

alexsnaps commented:
@maximebedard I addressed your concerns I think... and to you & @csfrancis I've added a somewhat more complete `README.md`, anything you'd want to see added?

Commit:
require
